Cargo.toml

```diff
@@ -32,7 +32,7 @@ std = [
 alloc = ["crossbeam-epoch/alloc", "crossbeam-utils/alloc"]
 
 [dependencies]
-cfg-if = "0.1.2"
+cfg-if = "~0.1.9"
```

Comment:
Personally, I prefer to downgrade the dependency version with ci instead of limiting the version. ([rayon did it before.](https://github.com/rayon-rs/rayon/commit/b21969416c36be0ad033452cebfd9a66db3da7f7#diff-354f30a63fb0907d4ad57269548329e3L20-L22))

Reply:
@taiki-e Would you be willing to submit a PR taking a similar strategy as in Rayon?

Reply:
The second commit of #428 is doing that: https://github.com/crossbeam-rs/crossbeam/pull/428/commits/b2e6cf83366512d1d0e369b757b855d95887e91a